packages/scaffold-ui/src/partials/w3m-connecting-wc-mobile/index.ts

```diff
@@ -90,6 +94,24 @@ export class W3mConnectingWcMobile extends W3mConnectingWidget {
 
   protected override onTryAgain() {
     if (!this.buffering) {
+      // Clear existing timeouts
```

Comment:
Seems like we have the same logics in the constructor, maybe we could move these business logics to a function and call it? Would increase the maintainability

Reply:
thank you - addressed